src/server_manager/web_app/ui_components/outline-region-picker-step.ts

```diff
@@ -25,17 +25,7 @@ import {COMMON_STYLES} from './cloud-install-styles';
 import {CloudLocationOption} from '../../model/location';
 import {getShortName, localizeCountry} from '../location_formatting';
 
-// TODO: Add more flags
 const FLAG_IMAGE_DIR = 'images/flags';
-const FLAG_MAPPING: {[countryCode: string]: string} = {
-  'IN': `${FLAG_IMAGE_DIR}/india.png`,
-  'SG': `${FLAG_IMAGE_DIR}/singapore.png`,
-  'UK': `${FLAG_IMAGE_DIR}/uk.png`,
-  'DE': `${FLAG_IMAGE_DIR}/germany.png`,
-  'NL': `${FLAG_IMAGE_DIR}/netherlands.png`,
-  'CA': `${FLAG_IMAGE_DIR}/canada.png`,
-  'US': `${FLAG_IMAGE_DIR}/us.png`,
-};
 
 @customElement('outline-region-picker-step')
 export class OutlineRegionPicker extends LitElement {
```

Comment:
Can you add a class to the subtitle and make it use the secondary text color and upper case?
This will make it closer to the mocks.

Reply:
Done